interface/patient_file/summary/stats_full.php

```diff
@@ -130,180 +130,177 @@ function newEncounter() {
+                                echo generate_display_field(array('data_type' => '1','list_id' => 'occurrence'), $row['occurrence']);
+                                echo "</td>\n";
+                                if ($focustype == "allergy") {
+                                    echo "  <td>";
+                                        echo generate_display_field(array('data_type' => '1','list_id' => 'reaction'), $row['reaction']);
+                                    echo "</td>\n";
+                                }
+                                if ($focustype == "allergy" || $focustype == "medical_problem") {
+                                    $codeListName = ($thistype == 'medical_problem') ? 'condition-verification' : 'allergyintolerance-verification';
+                                    echo "  <td>";
+                                        echo generate_display_field(array('data_type' => '1','list_id' => $codeListName), $row['verification']);
+                                    echo "</td>\n";
+                                }
+                                echo "  <td>" . text($row['referredby']) . "</td>\n";
+                                echo "  <td>" . text(oeFormatDateTime($row['modifydate'])) . "</td>\n";
+                                echo "  <td>" . text($row['comments']) . "</td>\n";
+                                echo "  <td id='e_" . attr($rowid) . "' class='noclick text-center' title='" . xla('View related encounters') . "'>";
+                                echo "  <button value='" . attr($ierow['count']) . "' class='btn btn-primary btn-sm editenc btn-edit' id='" . attr($rowid) . "'>" . xlt("Edit") . "</button>";
+                                echo "  </td>";
+                                echo " </tr>\n";
+                            }
                         }
-                    }
-
-                    // calculate the status
-                    if ($row['outcome'] == "1" && $row['enddate'] != null) {
-                          // Resolved
-                          $statusCompute = generate_display_field(array('data_type' => '1','list_id' => 'outcome'), $row['outcome']);
-                    } elseif ($row['enddate'] == null) {
-                          $statusCompute = xlt("Active");
-                    } else {
-                          $statusCompute = xlt("Inactive");
-                    }
-
-                    $click_class = 'statrow';
-                    if ($row['erx_source'] == 1 && $focustype == 'allergy') {
-                        $click_class = '';
-                    } elseif ($row['erx_uploaded'] == 1 && $focustype == 'medication') {
-                        $click_class = '';
-                    }
-
-                    echo " <tr class='" . attr($bgclass) . " detail' $colorstyle>\n";
-                    echo "  <td class='text-left " . attr($click_class) . "' style='text-decoration: underline' id='" . attr($rowid) . "'>" . text($disptitle) . "</td>\n";
```

Comment:
Looks like we are reverting changes from a recent commit here:
https://github.com/openemr/openemr/commit/4353d7c3aa5d2bc3b900496fb77bca3cf4667218